src/libs/actions/Report.js

```diff
@@ -1191,6 +1191,15 @@ function addAction(reportID, text, file) {
                 return;
             }
 
+            // Catch all errors
+            if (!response.reportAction) {
+                Growl.error(response.message);
+                Onyx.merge(`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${reportID}`, {
+                    [optimisticReportActionID]: null,
+                });
+                return;
+            }
+
```

Comment:
Sure. But I found a case when empty file is uploaded was .eps in my case ``Upload Error: No file content`` which keeps on loading if not handled.

Reply:
This should not be possible. We should block empty files whose size is 0 bytes.

Reply:
Let's continue the discussion here before I can tell you the next steps. OK. So the user should not be able to upload the empty files. Could you please test with other images with empty content?  Could you please confirm if those files have file.size `0`?

Reply:
Sure

Reply:
Yeah the size seems to be zero ``0`` checked on web.

Reply:
Great thanks. Ideally, that is a separate issue. But I don't mind those changes in this PR. @marcaaron Could you please confirm based on the following? 
1. This PR fixes the issues for SVG upload and blocks it from uploading. 
2. @Regaron found one issue where he is able to upload files with 0 sizes https://github.com/Expensify/App/pull/8021#discussion_r823547188. 

My intuition says that point 2 needs a new issue. What do you say? Sorry for the ping :bow:

Reply:
Hey, What is 0 sizes mean? I think we should most always be doing exactly what is stated in the linked issue. I would recommend not to do additional things as it will only delay approving the work you are hired to do. If you can summarize on the original issue how the scope has increased I will leave a comment there.

Reply:
If the file is created with empty content then it is being uploaded but the error thrown is not handled so the file is just showing loading.